iotlabmqtt/iotlabapi.py

```diff
@@ -187,9 +193,18 @@ def node_command(self, command, cmd_opt, archi, *nums):
 
     def _node_command(self, command, cmd_opt, archi, *nums):
         import iotlabcli.node
+        import iotlabsshcli.open_a8
         nodes = self._nodes_for_num(archi, *nums)
-        result = iotlabcli.node.node_command(self.api, command, self.expid,
-                                             nodes, cmd_opt)
+
+        if archi == 'm3':
+            result = iotlabcli.node.node_command(self.api, command, self.expid,
+                                                 nodes, cmd_opt)
+        elif archi == 'a8':
+            # NOTE: Open-a8-node use different syntax and dictionary format
+            config_ssh = {'user': self.user, 'exp_id': self.expid}
+            result_ = iotlabsshcli.open_a8.flash_m3(config_ssh, nodes, cmd_opt)
```

Comment:
Note that, "ssh <site>.iot-lab.info" might not work when ran on "<site>.iot-lab.info" (need to have the public key in known_hosts), which causes this line to fail.